ui/app/AppLayouts/Chat/ChatColumn/ChatButtons.qml

```diff
@@ -136,6 +147,50 @@ Item {
         }
     }
 
+    Rectangle {
```

Comment:
We'll need to make this a reusable component at some point since we just copy the same base stuff and change the icon and action